.github/workflows/orchestrator-autofix.yaml

```
@@ -1,25 +1,51 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 
 jobs:
-  auto-fix:
-    name: 🔧 Auto-Fix Everything
+  autofix:
+    name: 🔧 Auto-Fix & Format
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      workflows: write
```

actionlint (1.7.9)

unknown permission scope "workflows". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)